src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/NativeRuntimeEventSource.cs

```diff
@@ -7,14 +7,16 @@ namespace System.Diagnostics.Tracing
     /// NativeRuntimeEventSource is an EventSource that represents the ETW/EventPipe events emitted by the native runtime.
     /// Most of NativeRuntimeEventSource is auto-generated by scripts/genRuntimeEventSources.py based on the contents of the Microsoft-Windows-DotNETRuntime provider.
     /// </summary>
-    [EventSource(Guid = "5E5BB766-BBFC-5662-0548-1D44FAD9BB56", Name = "Microsoft-Windows-DotNETRuntime")]
+    [EventSource(Guid = "5E5BB766-BBFC-5662-0548-1D44FAD9BB56", Name = EventSourceName)]
+    [EventSourceAutoGenerate]
     internal sealed partial class NativeRuntimeEventSource : EventSource
     {
         internal const string EventSourceName = "Microsoft-Windows-DotNETRuntime";
         internal static NativeRuntimeEventSource Log = new NativeRuntimeEventSource();
 
-        // The NativeRuntimeEventSource GUID is {5e5bb766-bbfc-5662-0548-1d44fad9bb56}
-        private NativeRuntimeEventSource() : base(new Guid(0x5e5bb766, 0xbbfc, 0x5662, 0x05, 0x48, 0x1d, 0x44, 0xfa, 0xd9, 0xbb, 0x56), EventSourceName) { }
+        // Parameterized constructor to block initalization and ensure the EventSourceGenerator is creating the default constructor
```

Comment:
typo, initialization (various places)

Reply:
Dangers of artisanal small batch PRs 😉

Fixed